src/gui/folder.cpp

```diff
@@ -64,6 +64,18 @@ auto versionC()
 
 constexpr int WinVfsSettingsVersion = 4;
 constexpr int SettingsVersion = 2;
+
+// Settings keys:
+const char localPathC[] = "localPath";
+const char journalPathC[] = "journalPath";
+const char targetPathC[] = "targetPath";
+const char pausedC[] = "paused";
+const char ignoreHiddenFilesC[] = "ignoreHiddenFiles";
```

Comment:
Could you directly make them QStrings/QLatin1Strings here?

Reply:
done